Library/src/main/java/com/shopify/testify/internal/exception/ScreenshotBaselineNotDefinedException.kt

```diff
@@ -24,5 +24,16 @@
 
 package com.shopify.testify.internal.exception
 
-class ScreenshotBaselineNotDefinedException(moduleName: String, testName: String, testClass: String) :
-        Exception("\n\n*  A baseline screenshot could not be found for '$testName'.\n*  To record a baseline screenshot, run `./gradlew $moduleName:screenshotRecord -PtestClass=$testClass`\n")
+class ScreenshotBaselineNotDefinedException(
+    moduleName: String,
+    testName: String,
+    testClass: String,
+    deviceKey: String
+) :
+        Exception(
+                "\n" +
+                "\n*  A baseline screenshot could not be found for '$testName'." +
+                "\n*  Baseline could not be found in $deviceKey" +
+                "\n*  To record a baseline screenshot, run `./gradlew $moduleName:screenshotRecord -PtestClass=$testClass`" +
+                "\n"
```

Comment:
nit: I think that you could use a raw string here to help you format the text without concatenating multiple strings.
https://kotlinlang.org/docs/reference/basic-types.html#string-literals

Reply:
Yes, you're probably right but I find raw strings looks crappy.
Anyway, I'll change it